apps/src/blockly/addons/cdoTrashcan.ts

```diff
@@ -57,9 +57,10 @@ export default class CdoTrashcan extends GoogleBlockly.DeleteArea {
     this.workspace.getComponentManager().addComponent({
       component: this,
       // Weight determines the order of drag targets. The toolbox is also a drag
-      // target (weight 1). onDragEnter/Exit/Over are only called for the first
+      // target (weight 0). onDragEnter/Exit/Over are only called for the first
```

Comment:
-1?

Reply:
This comment is about the Toolbox, which is now 0. The trashcan is -1 so that it takes precedence over the toolbox.

Reply:
ah gotcha, nvm!